libs/zephyr-agent/src/lib/build-context/ze-util-get-git-info.ts

```diff
@@ -57,25 +57,29 @@ async function loadGitInfo(hasSecretToken: boolean) {
     'git rev-parse --abbrev-ref HEAD',
     'git rev-parse HEAD',
     'git tag --points-at HEAD',
+    // Get the current checked out tag
+    "git reflog -1 --pretty=format:'%D' | grep -o 'tag: [^ ,]*' | sed 's/tag: //'",
```

Comment:
as far as I know this will fail at CI
`git tag --points-at HEAD | paste -sd "," -` will not fail
 output sample in screen
![image](https://github.com/user-attachments/assets/d470f946-390a-4c19-a948-777eb647bb91)